tdrs-frontend/src/assets/IdleTimer.scss

```diff
@@ -1,12 +1,11 @@
 .modal {
   position: fixed; 
-  z-index: 1; 
+  z-index: 1000; 
```

Comment:
@jorgegonzalez can you say something about why this change was made?

Reply:
It is not clear to me why this was needed as I didn't make this change. Reverting it locally, I didn't notice any differences in the IdleTimer modal. Should we reverse this?

Reply:
It may have been something related to accessibility, but I'm not sure since there are no comments. You might want to reach out to @reitermb to see if there was an issue surrounding this object

Reply:
I know we went through several iterations on the modal to check all the a11y boxes, but I'm not totally sure if the z-index here was one of those. 

If we change it I'd want to test again with a screenreader to make sure focus is still behaving as it should. E.g., Modal gets announced when it appears, focus remains inside it when tabbing around, focus goes back to where it was before the modal opened when it's closed.

My biggest question with this change would be whether it disrupts the focus going back to where it was bit.

Reply:
Ok - I think we should undo the change and redeploy the branch and check a11y. Then if we have to change it back we can add a comment indicating why the change was necessary